probe_src/python/workflow/nextflow.md

```diff
@@ -0,0 +1,40 @@
+# Nextflow
+Nextflow is a powerful workflow management system that facilitates the writing of computational pipelines.
+
+## Installation
+### Prerequisites
+Java 8 or later must be installed on your system.
```

Comment:
Let's use flake.nix to install Nexflow instead. This will ensure a consistent version of Nextflow and JRE as well. Add `pkgs.nextflow` to `flake.nix` and run `nix develop`.